usersync/cookie.go

```diff
@@ -176,6 +176,9 @@ func (cookie *PBSCookie) GetId(bidderName openrtb_ext.BidderName) (id string, ex
 
 // SetCookieOnResponse is a shortcut for "ToHTTPCookie(); cookie.setDomain(domain); setCookie(w, cookie)"
 func (cookie *PBSCookie) SetCookieOnResponse(w http.ResponseWriter, setSiteCookie bool, cfg *config.HostCookie, ttl time.Duration) {
+	if !cfg.Refresh {
+		ttl = cookie.birthday.Add(ttl).Sub(time.Now())
+	}
```

Comment:
Please add a test case for this new condition.

Reply:
They decided that cookie refresh is forbiden whatever the user consent might have been and regardless how the vendor declared himself in the TCF.

Reply:
I've added a test case, hope it's what was expected.